jsonstreams/__init__.py

```diff
@@ -144,7 +144,10 @@ def __init__(self, fd, indent, baseindent, encoder, pretty):
             self.write_comma_literal = functools.partial(self.raw_write, ', ')
 
     def _indent(self):
-        return ' ' * self.baseindent * self.indent
+        if self.indent and isinstance(self.indent, int) and self.indent >= 0:
+            return ' ' * self.baseindent * self.indent
+        else:
+            return ''
```

Comment:
I've really been meaning to getting around to fixing the type annotations, but let's just assume that only an int or None is valid for self.indent. In that case I think just
```python
if self.indent:
   ...
...
```
is fine.